features/items.md

```diff
@@ -167,18 +184,17 @@ The output will be
     Window is 1
     Window is opened
 
-Transform file use ISO-8859-1 encoding. If you want your accent and special character to show in the right way, you need to precise this encoding in your editor. If you must use UTF-8, you will need to escape the character. See [UTF-8 encoding table](http://www.utf8-chartable.de/)
-
-Example: é will be written ```\u00E9```
+Transform files use UTF-8 encoding, so unicode symbols will also work.
```

Comment:
Regarding your other question: Yes these can be used in Transformations, Items, Rules and Sitemaps ;)